pyvista/core/filters/data_set.py

```diff
@@ -2044,7 +2044,7 @@ def elevation(  # type: ignore[misc]
 
     def contour(  # type: ignore[misc]
         self: ConcreteDataSetType,
-        isosurfaces: int = 10,
+        isosurfaces: int | Sequence[float] = 10,
```

Comment:
Can we use `VectorLike[float]` here? (to match typing for other vector-like inputs)
```suggestion
        isosurfaces: int | VectorLike[float] = 10,
```